public/components/app.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
there is some issue to use BrowserRouter
in case observers some  route logic doesn't meet expectation,  while adding more page, and more complex navigation between page. 

we may switch to HashRouter, not require to change, just FYR

Reply:
Will switch- was using Hash router in old platform and don't want to incur any avoidable tech debt